aws/policy/compute.yaml

```diff
@@ -136,7 +136,11 @@ Statement:
       - lambda:CreateFunction
       - lambda:DeleteFunction
       - lambda:GetFunction
+      - lambda:GetFunctionConfiguration
       - lambda:GetPolicy
+      - lambda:ListVersionsByFunction
+      - lambda:ListAliases
+      - lambda:ListEventSourceMappings
```

Comment:
The List* actions need to be added at line 62, where they would be applied to `Resource: "*"`. This block only affects `arn:aws:lambda:us-east-1:{{ aws_account_id }}:function:*` which is insufficient for the list functions.

I have tested 64548 against a policy that has these list function in that section, and everything runs with that change.

Reply:
Done